src/main/java/frc/team2412/robot/util/AutoLogic.java

```diff
@@ -0,0 +1,79 @@
+package frc.team2412.robot.util;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.auto.NamedCommands;
+import com.pathplanner.lib.path.PathPlannerPath;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.team2412.robot.Controls;
+import frc.team2412.robot.Robot;
+import frc.team2412.robot.Subsystems;
+import frc.team2412.robot.commands.intake.*;
+
+public class AutoLogic {
+	public Robot r = Robot.getInstance();
+	private final Subsystems s;
+	private final Controls controls;
+
+	private SequentialCommandGroup vibrateControllerCommand;
+
+	public boolean dummyLogic() {
+		return true;
+	}
```

Comment:
What's this method for?

Reply:
placeholder while waiting for limelight pr (need method for determining if note is present where were looking at)